tests/kuttl/tests/dataplane-deploy-nodehashes-test/00-assert.yaml

```diff
@@ -7,8 +7,6 @@ spec:
   - edpm-compute-no-nodes
 status:
   deployed: true
-  nodeSetHashes:
-    edpm-compute-no-nodes: n595h74h98h5d8h579h5c7h84h5dh599h544hf8h65bh686h5f7h99h669h676h97h678h646h9fh5cch649h66fhcch5fh688h95h7bhbh675hd9q
 ---
```

Comment:
I pushed a commit to also drop the above to your branch, hope you don't mind. that should fix the kuttl tests here hopefully.